ashley/README.md

```diff
@@ -0,0 +1,83 @@
+# KTX: `Ashley` entity component system utilities
+
+Utilities and type-safe builders for the [Ashley](https://github.com/libgdx/ashley) entity component system.
+
+### Why?
+We can make using [Ashley](https://github.com/libgdx/ashley) with Kotlin more pleasant where Java Classes are 
+required for many method calls by providing helper methods using reified types. Additionally, creating Entities and 
+their respective Components can result in a lot of declarative-style code which is suited nicely to a readable type-safe 
+builder DSL. 
+ 
+### Guide
+
+`ktx-ashley` provides extensions and utilities for using [Ashley](https://github.com/libgdx/ashley) with Kotlin:
+
+- `Engine.add` and `Engine.entity` extension methods provide type-safe building DSL for creating non-pooled Entities
+- `PooledEngine.add` and `PooledEngine.entity` extension methods provide type-safe building DSL for creating pooled Entities
```

Comment:
You can warn the users about `PooledEngine` usage - if refer to it by interface (as an `Engine`), you might not use the pooling functionality after all.